test.cpp

```diff
@@ -66,8 +66,9 @@ TEST(Parser, ParseStrError) {
   ASSERT_TRUE(result.hasError());
   ASSERT_EQ(result.getError(), "Expected string on row 1, col 1");
 }
-
-TEST(Parser, ParseOut) {
+// TODO(PiJoules): Implement peak in the lexer, so we don't lex the out token by
+// the time we hit ParseOut
+TEST(Parser, DISABLED_ParseOut) {
```

Comment:
If we merge https://github.com/dsLeks/XOS-Language/pull/42 first, we can use peeking and don't have to disable this test.